packages/rest/src/rest.server.ts

```diff
@@ -987,6 +988,37 @@ export class RestServer extends BaseMiddlewareRegistry
   ): void {
     this._externalRoutes.mountRouter(basePath, router, spec);
   }
+
+  /**
+   * Export the OpenAPI spec to the given json or yaml file
+   * @param outFile - File name for the spec. The extension of the file
+   * determines the format of the file.
+   * - `yaml` or `yml`: YAML
+   * - `json` or other: JSON
+   * If the outFile is `''` or `'-'`, the spec is written to the console.
+   * @param log - Log function, default to `console.log`
+   */
+  async exportOpenApiSpec(
+    outFile: string,
+    log = console.log,
+  ): Promise<OpenApiSpec> {
+    const spec = await this.getApiSpec();
+    if (outFile === '-' || outFile === '') {
```

Comment:
What if `outFile` is not provided at all (equals to `undefined`)? Can we please treat that case as "print to stdout" too?

Reply:
Fixed with a default value.